test/noLifecycleCallRule.spec.ts

```diff
@@ -1,51 +1,46 @@
-import { lifecycleHooksMethods, LifecycleHooksMethods, Rule } from '../src/noLifeCycleCallRule';
+import { Rule } from '../src/noLifecycleCallRule';
```

Comment:
Same here. I see we've merged the template string snippets, but I have no idea what we are testing like this. Would you mind updating? Thanks 🙇🏼‍♂️

Reply:
No problem, I'll update ASAP.